packages/runtime/src/errors/errors-list.ts

```diff
@@ -8,291 +9,297 @@ export interface ErrorDescriptor {
   description: string
 }
 
-export function getErrorCode (error: ErrorDescriptor): string {
-  return `${ERROR_PREFIX}${error.number}`;
+export function getTealErrorCode (error: ErrorDescriptor): string {
+  return `${TEAL_ERROR_PREFIX}${error.number}`;
 }
 
-export const ERROR_RANGES = {
-  TEAL: { min: 0, max: 99, title: "TEAL opcode errors" },
-  TRANSACTION: { min: 100, max: 199, title: "Transaction error" },
-  ASA: { min: 900, max: 999, title: "ASA File Parse Error" }
+export function getRuntimeErrorCode (error: ErrorDescriptor): string {
+  return `${RUNTIME_ERROR_PREFIX}${error.number}`;
+}
+
+export const TEAL_ERROR_RANGES = {
+  TEAL: { min: 1000, max: 1099, title: "TEAL opcode errors" }
```

Comment:
What do you think about renaming this error to `EXE` title: "TEAL execution error" ?

Reply:
Agree with title change. Did you mean to rename the name as well (`TEAL` to `EXE`)?

Reply:
good idea, changing TEAL to  EXE